staging/src/kubesphere.io/api/types/v1beta1/federatedingress_types.go

```diff
@@ -45,7 +45,7 @@ type FederatedIngressSpec struct {
 }
 
 type IngressTemplate struct {
-	Spec extensionsv1beta1.IngressSpec `json:"spec,omitempty"`
+	Spec networkingv1.IngressSpec `json:"spec,omitempty"`
```

Comment:
@yuswift @wenhuwang 
Needs to consider FederatedIngress API version conversion. Build-in Ingress resources could be handled with Kube-apiserver by default. But not for CRD.

Reply:
We don't use the file to generate crd files. So I think we can skip it.

Reply:
Subsequent ingress will use the network.k8s.io/v1 version, I think IngressTemplate should also be upgraded together

Reply:
Yes, it should be upgraded.